site/app/uploadcare/customStorage.js

```diff
@@ -0,0 +1,44 @@
+class CustomStorage {
+  // S3 custom storage URLs are defined in Uploadcare as
```

Comment:
This comment is supposed to go with the STORAGE_ALIAS_PREFIX declaration below.

Reply:
done